sbd.spec

```diff
@@ -21,6 +21,15 @@
 %global github_owner beekhof
 %global buildnum 1
 
+%ifarch s390x s390
+%global watchdog_timeout_default 15
```

Comment:
maybe worth a comment why this is

Reply:
this differentiation has always been there without comments. but you are right not everybody has a mainframe to verify. commented.